.cfignore

```diff
@@ -9,3 +9,8 @@ cg-deck
 cg-dashboard
 /screenshots/
 static/tests/
+.org.chromium*
+# only exclude these index.html files from the VNC service.
+# they only appear in the root folder.
+/index.html*
```

Comment:
I see that we are ignoring these index.html files, but running the commands specified in the docker README for the first time generated something like 250 index.html files. Is there a way we can reduce that number?

Or are those all safe to delete? If so, maybe we could add a script to the package.json to manage that?